app/models/symphony/pickup_destination.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Symphony
+  # Encapsulates pickup destination logic for Symphony
+  class PickupDestination
+    # Get the default destination
+    def self.default_destination
+      Settings.default_pickup_library
+    end
+
+    # @param code: service point or library code
+    def initialize(destination_code)
+      @code = destination_code
+      @type = 'Library'
```

Comment:
I don't see this used. Do we need it?

Reply:
If you're referring to @type - it didn't get used so I removed it.